docs/reference/commandline/ps.md

```diff
@@ -51,7 +51,7 @@ The currently supported filters are:
 * exited (int - the code of exited containers. Only useful with `--all`)
 * status (created|restarting|running|paused|exited)
 * ancestor (`<image-name>[:<tag>]`,  `<image id>` or `<image@digest>`) - filters containers that were created from the given image or a descendant.
-* isolation (default|hyperv)   (Windows daemon only)
+* isolation (default|process|hyperv)   (Windows daemon only)
```

Comment:
OK, I'm not that bothered either way. Removed from docs above. I'll open a code change PR for changing the filter code shortly (as this PR is a key one for TP4 still)